introduction.md

```diff
@@ -1,34 +1,79 @@
-# Week 01 - Introduction 
+# Week 01 - Introduction
 
-### Goals 
-- Gain insights about the kids, 
+### Goals
+- Gain insights about the kids,
 - Establish context for the program,
 - Inspire them, to look forward for the next session
 
-### Survey 
+### Survey
 - Should be fun to engage in
 - Gain insights on prior knowledge, interests, expereinces, behaviour
 #### Questions
 - What comes to your mind when we say "computers" or "technology"?
 - How many of you have used a computer or phone or calculater in the past?
-- Have you ever wondered how they work or thought about building something like them or doing something with them? 
-- What do you want to become in your life? 
+- Have you ever wondered how they work or thought about building something like them or doing something with them?
+- What do you want to become in your life?
 
-- We should check if our upcoming expames and analogies are relavent to the kids or not. 
+- We should check if our upcoming expames and analogies are relavent to the kids or not.
 
-### Morning Routine 
-Description : An unplugged exercise to show the kids what an algorithm or generic code looks like, by breaking down their morning 
-routine of getting up from bed and going to school. We can introduce code structure, flow, if statements, loops, through this exercise 
+Description : An unplugged exercise to show the kids what an algorithm or generic code looks like
 
-### History of CS 
-Description : Walk the kids through the history of computers, to answer questions like "why were they invented", 
-"what problems do they solve", "why do we need them", "how did they evolve over time". The goal here is to establish the context of why 
-computers exist in the first place, but in a very story-like fashion. 
+Goal: A metaphor that we can map back to coding.
+We can introduce code structure, flow, if statements, loops, through this exercise and keep getting back to this example throughout the course.
+
+For week1, we can stop with using this to introduce the concept of `algorithm`, and maybe
+`code` and `programming language`.
+
+*Example#1*
+- Breaking down their morning routine of getting up from bed and going to school
+What we expect more or less - get up => brush => shower =>
+breakfast => wait for the right bus (loops/conditionals) => get to class =>
+greet your friends (function concept), etc
+
+*Example#2*
+- Planning a birthday party for a friend. <TODO: @Madhu94 add a short description>
+
+*Example#3* (From code.org)
```

Comment:
Is the suggestion to consider all three of these examples for the Introduction session? Or to uses different examples throughout the program to introduce and reintroduce the concepts of programming to the kids? If the later, then we don't need to mention the other two examples (birthday party and planting a tree) in this document

Reply:
I actually did think we could try out more than one example, just for re-inforcement/application of the "algorithm" concept. Do you think we won't have the time? We can remove the other two then

Reply:
Yeah, it's not just about the time, even though that is a factor. I just though that one elaborate well executed example would be sufficient. We could have another example, but we can try to make the kids break it down into steps, instead of us

Reply:
Added a note that this could be for homework or maybe a workbook exercise, if we decide to have those